include/snowflake/sb_CRTFunctionSafe.h

```diff
@@ -1,13 +1,9 @@
-// =================================================================================================
-///  @file Simba_CRTFunctionSafe.h
-///
-///  C Run-Time library functions' definitions for cross platform use.
-///
-///  Copyright (C) 2019 Simba Technologies Incorporated.
-// =================================================================================================
+/*
+ * Copyright (c) 2019-2024 Snowflake Computing, Inc. All rights reserved.
```

Comment:
is it right to change the copyright here? if so then we could name the file with `sf_` prefix not `sb_`?

changing the file is breaking compatibility and we should consider it as a breaking change and release next version with bumped major part

Reply:
No change on this for now handle it in separated issue/PR